repository-data/application/src/main/resources/hcm-content/configuration/update/registry-1.yaml

```diff
@@ -0,0 +1,30 @@
+---
+/hippo:configuration/hippo:update/hippo:registry/Clean Hippo Mirror Docbase Values Having Paths:
+  hipposys:batchsize: 10
+  hipposys:description: A example script to clean interim hippo:docbase values of
+    Hippo Mirror nodes having paths instead of UUIDs.
+  hipposys:dryrun: false
+  hipposys:parameters: '{}'
+  hipposys:query: //element(*)[jcr:like(@hippo:docbase,'/content/%')]
+  hipposys:script: "/*\r\n * Copyright 2015-2017 Hippo B.V. (http://www.onehippo.com)\r\
```

Comment:
can you add it the same way the other scripts are added - makes it easy to ready

like
https://github.com/NHS-digital-website/hippo/blob/master/repository-data/application/src/main/resources/hcm-config/configuration/update/MigratorImporterScript.yaml#L24-L26

https://github.com/NHS-digital-website/hippo/blob/master/repository-data/application/src/main/resources/hcm-config/configuration/update/MigratorImporterScript.groovy

Reply:
Good point! Thanks